.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "src/v/googletest"]
+	path = src/v/googletest
+	url = https://github.com/google/googletest
```

Comment:
Any reason for making this a submodule? While I do like submodules (possibly the opposite to many people) I am surprised we now have submodules for some deps but the cmake download way for others?

Reply:
hmm, yeh just testing the waters. no love for the cmake superbuild but yeh maybe better to not mix them right now.

Reply:
@StephanDollberg I fixed this. The companion PR in vtools for dependencies is here https://github.com/redpanda-data/vtools/pull/2010